packages/Babylonian-Printbugger.package/BPPrintbugger.class/instance/addExample..st

```diff
@@ -0,0 +1,7 @@
+private
+addExample: aBPExample
+	
+	(BPExamples value activeExamples size == 1) ifTrue: [canvasMorph removeAllMorphs].
```

Comment:
Just out of interest: Why are all Morphs deleted when the size is 1? :)

(Also use #= instead of #==)

Reply:
The check removes the "no active examples have been found to display" since =1 means the first example is now active